api/scpca_portal/management/commands/create_portal_metadata.py

```diff
@@ -20,11 +20,15 @@ def add_arguments(self, parser):
         parser.add_argument(
             "--clean-up-output-data", action=BooleanOptionalAction, default=settings.PRODUCTION
         )
+        parser.add_argument("--delete-from-s3", action=BooleanOptionalAction, default=False)
         parser.add_argument(
             "--update-s3", action=BooleanOptionalAction, default=settings.UPDATE_S3_DATA
         )
+        parser.add_argument("--purge", action=BooleanOptionalAction, default=False)
 
     def handle(self, *args, **kwargs):
+        if kwargs("purge", False):
+            self.purge_computed_file(delete_from_s3=kwargs.get("delete_from_s3", False))
         self.create_portal_metadata(**kwargs)
 
     def create_portal_metadata(self, **kwargs):
```

Comment:
here you would add:

```python
if computed_file := ComputedFile.objects.filter(portal_metadata_only=True).first():
    self.purge_computed_file(computed_file, delete_from_s3)

```

Reply:
See at 9b19c58cad3b77feaafca2d726224d13a8c6d932.